File: 3783-mirror-distance-of-an-integer/3783-mirror-distance-of-an-integer.java
class Solution {
    public int mirrorDistance(int n) {
        int n2=n;
        int rev=0;
        while(n2!=0){
            int l=n2%10;
            rev=rev*10+l;
            n2=n2/10;
        }
        return Math.abs(n-rev);
    }
}

File: 3783-mirror-distance-of-an-integer/3783-mirror-distance-of-an-integer.cpp
class Solution {
public:
    int mirrorDistance(int n) {
        int a=n;
        int md=0;
        int prd=0;
        while(a>0){
            md=a%10;
            prd=prd*10 + md;
            a=a/10;
        } 
        return abs(n-prd);
    }
};